Formula/embulk.rb

```diff
@@ -8,12 +8,10 @@ class Embulk < Formula
 
   depends_on :java
 
-  skip_clean "libexec"
-
   def install
-    (libexec/"bin").install "embulk-#{version}.jar" => "embulk"
-    chmod 0755, libexec/"bin/embulk"
-    bin.write_exec_script libexec/"bin/embulk"
+    # Execute through /bin/bash to be compatible with OS X 10.9.
+    libexec.install "embulk-#{version}.jar" => "embulk.jar"
+    (bin/"embulk").write "#!/bin/bash\nexec /bin/bash \"#{libexec/"embulk.jar"}\" \"$@\"\n"
```

Comment:
Updated to use EOS :-)
`Pathname#write_jar_script` doesn't work well for embulk because the init script of embulk sets (some parameters)[https://github.com/embulk/embulk/blob/master/embulk-cli/src/main/sh/selfrun.sh#L150-L154] to java. Those parameters are important especially for shorter startup time.